src/components/applicationForm/contactInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormHeader, FormQuestion } from "../formSections"
+
+const questions = [
```

Comment:
Currently, Im thinking of:

Example:
{
question_text: "First name"
question_type: 1
is_required: true
order: 1
},
{
question_text: "Age"
question_type: 3
options: [ "18-21", "22-25", ">25" ]
is_required: false
order: 5
},
{
question_text: "skills"
description: "What kind of tech skills you are comfortable with"
question_type: 5
options: ["rails", "react", "node"] 
is_required: false
order: 2
}

Reply:
If i am not wrong, there should be these categories of question_type:
text_short  --- 1
text_long --- 2
dropdown --- 3
single_select (mcq) --- 4
multi_select (mrq) --- 5
datefield --- 6

question_type should be encoded as a number.

The json should contain something like this:
ID: A unique identifier for the question.
Text: The main question text.
Description: Placeholder or helper text for context.
Type: Encoded type (e.g., text, multiple-choice, dropdown).
Options: For non-text questions, choices are stored as an array.
Validation: Whether the question is required, max/min inputs, etc.
Order: Determines the position in the form.

What do you think?

Reply:
Honestly, that looks pretty neat! 
Slight correction, datefield has 2 types, singular dates and start-end dates. Easily extensible. 

Still, aside from `is_required`, I'm not so sure on what other types of validation are needed. Max/min inputs are currently handled per component, i.e. short answers are limited to 50 characters and long answers to 200 characters, etc. Glad for the discussion!